uber/templates/preregistration/dealer_registration.html

```diff
@@ -1,10 +0,0 @@
-{% extends "./preregistration/preregbase.html" %}
```

Comment:
The ``dealer_registration`` page handler doesn't actually render this template, instead it returns the result of calling ``self.form``, which makes this template unnecessary.